### ISSUE TITLE
Push API for AlpAkka Google Cloud Pub Sub

### ISSUE THREAD

**Issue by vertexclique** [akka/alpakka#826]
There is push API for Google Cloud Pub-Sub. Can we include this in the roadmap aswell?

**Comment by francisdb** [comment]
Currently the pub-sub integration is focused on streaming and not a full pub-sub api client.
So I guess you would like to see all other api calls implemented in a non-blocking fashion as that is not available on the official google java client?

**Comment by vertexclique** [comment]
Yes, since we are not using official Java client in this project, we can implement pull interface also.
